packages/vue/src/components/FRadioField/FRadioField.spec.ts

```diff
@@ -15,8 +15,7 @@ function createWrapper({
         slots: { default: "Default label", ...slots },
         global: {
             provide: {
-                sharedName: "providedName",
-                [getFieldsetLabelText]: () => undefined,
+                [fieldsetInjectionKeys.sharedName as symbol]: "providedName",
```

Comment:
I think this is bad practice but didn't want to rewrite the test-cases: it should rather actually wrap the component inside a `<f-fieldset name="providedName">` instead.